tfutils/model.py

```diff
@@ -196,7 +196,7 @@ def dropout(self, dropout=.5, in_layer=None):
 
 def mnist(inputs, train=True, **kwargs):
     m = ConvNet(**kwargs)
-    reuse = None if train else True
+    reuse = None #if train else True
```

Comment:
I did this because in the original version (reuse = None if train else True), no containing scope would reset reuse, which it needs to.   Sometimes even when you're testing, you variables aren't being reused -- e.g. the first time you re-start the model for testing.   Maybe I don't understand your use case.  Let's talk about this over chat.